python/veles/async_client/node.py

```diff
@@ -0,0 +1,392 @@
+# Copyright 2017 CodiLime
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+
+from veles.async_conn.node import AsyncNode
+from veles.proto import messages
+from veles.proto.node import PosFilter
+from veles.schema.nodeid import NodeID
+
+from veles.proto.exceptions import (
+    SchemaError,
+)
+
+
+def _resolve_obj(conn, obj):
```

Comment:
We already have identical function in other `node.py` in other module - maybe we should create some module with common utility functions?